src/RobustAdaptiveMetropolis.jl

```diff
@@ -53,154 +43,243 @@ julia> # Number of warmup steps, i.e. the number of steps to adapt the covarianc
        # by default in the `sample` call. To include them, pass `discard_initial=0` to `sample`.
        num_warmup = 10_000;
 
-julia> # Set the seed so get some consistency.
-       Random.seed!(1234);
-
 julia> # Sample!
-       chain = sample(model, RAM(), 10_000; chain_type=Chains, num_warmup=10_000, progress=false, initial_params=zeros(2));
+       chain = sample(
+            model,
+            RobustAdaptiveMetropolis(),
+            num_samples;
+            chain_type=Chains, num_warmup, progress=false, initial_params=zeros(2)
+        );
+
+julia> isapprox(cov(Array(chain)), model.Σ; rtol = 0.2)
+true
+```
+
+It's also possible to restrict the eigenvalues to avoid either too small or too large values. See p. 13 in [^VIH12].
+
+```jldoctest ram-gaussian
+julia> chain = sample(
+           model,
+           RobustAdaptiveMetropolis(eigenvalue_lower_bound=0.1, eigenvalue_upper_bound=2.0),
+           num_samples;
+           chain_type=Chains, num_warmup, progress=false, initial_params=zeros(2)
+       );
 
 julia> norm(cov(Array(chain)) - [1.0 0.5; 0.5 1.0]) < 0.2
 true
-```
+````
```

Comment:
That looks like one backtick too many. PS. How do I make a GitHub suggestion for this?

Reply:
`````suggestion
````
`````

Reply:
On second thought, I just pushed a fix for this myself.

Reply:
Thanks @penelopeysm. :) I should have guessed that the universe would have been organised in such a way that the fix to quadruple backticks would be more quadruple backticks.